### DISCUSSION TITLE
Unable to copy items into start of expedition & Strange Expedition Badge

### DISCUSSION THREAD

**Discussion by GitFo0** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#8] [General]
Hi, 

Thanks for putting the time into this, really cool and makes a single player game kinda guy happy.

I run an GOG offline installer copy of NMS, if that's of any relevance (Worlds Update), and I've got a problem copying items over when starting an expedition, it shows my inventory as empty, same for my ships inventory when clicking on a space to add an item to at the expedition terminal on the Anomaly. 

If its of any relevance, I store the cache json here: 
`<driver-letter>:\Users\<user-name>\AppData\Roaming\HelloGames\NMS\DefaultUser\cache`

Also when looking at the `Catalogue & Guides -> Previous Expeditions`, from my main save, I've got an extra expedition badge for an expedition I haven't completed (the one I'm currently doing), its the icon for `#1 The Pioneers`, but it's name is "Season 1 Name"

Having said all this, I could start the expedition fine, I'm half way through it now, hopefully there won't be an issue copying my goodies back to my main save when its done.

**Comment by GitFo0** [comment]
Heres a screenshot of my extra badge, this is the expedition I'm currently running from you're cache files.

![IMG_20240724_193032_516](https://github.com/user-attachments/assets/ac5e049c-e70d-49c7-ab06-044cf3c39414)

**Comment by GitFo0** [comment]
I suppose its possible these are a result of doing expeditions on an offline install copy instead of an galaxy copy and going offline.

**Reply by cwmonkey** [reply]
Hm yeah, neither of us has played using an offline install copy. If you have any issues, try grabbing the json from mys site: https://cwmonkey.github.io/nms-expeditions/

There shouldn't be any differences at this point, but that seems to work for some people for some reason.

**Reply by IsAmDead** [reply]
I have the same issue with the badge as well. I’m not using an offline install copy.
